src/diet.ts

```diff
@@ -316,6 +316,8 @@ function menu(): MenuItem<Note>[] {
    *  > js Item.all().filter((item) => item.fullness > 0 && item.name.indexOf("lasagna") > 0 && getIngredients(item)["savory dry noodles"]).join(", ")
    */
   const lasagnas = $items`fishy fish lasagna, gnat lasagna, long pork lasagna`;
+
```

Comment:
can we add a comment noting what standardSpleenItem means, something like:
``` // standardSpleenItem indicates a spleen item of size 4 with an adventure yield of 5-10. They are functionally equivalent.```

Reply:
If you make that a formal suggestion I can commit it right now otherwise it'll have to wait until I'm at my computer